packages/stryker-mocha-runner/src/MochaTestRunner.ts

```diff
@@ -82,7 +95,7 @@ export default class MochaTestRunner extends EventEmitter implements TestRunner
   }
 
   private additionalRequires() {
-    if (this.mochaRunnerOptions && this.mochaRunnerOptions.require) {
```

Comment:
Are we sure `this.mochaRunnerOptions` is a valid object here instead of undefined?

Reply:
Yes, because we know all config editors are loaded and the mocha config editor does this:

`config[mochaOptionsKey] = new MochaOptionsLoader().load(config);`

It is not ideal, but at the same time, i don't want to keep rechecking that variable to be defined.